src/convertToObject.js

```diff
@@ -12,7 +12,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedToObject = {};
+
+  const arrayOfProperties = sourceString.split(';').map(string => {
+    return string.trim().split(':');
+  });
+
+  for (const element of arrayOfProperties) {
+    if (element[0] === '') {
+      continue;
+    }
+
+    convertedToObject[`${element[0].trim()}`] = element[1].trim();
```

Comment:
```suggestion
    convertedToObject[element[0].trim()] = element[1].trim();
```
Зачем? У тебя там и так строка